app/reducers/events.ts

```diff
@@ -208,7 +208,9 @@ export const selectUpcomingEvents = createSelector(selectEvents, (events) =>
   events.filter((event) => event.isUsersUpcoming)
 );
 export const selectSortedEvents = createSelector(selectEvents, (events) =>
```

Comment:
This might be out of scope, but could it be possible to add some types to these functions, to make it easier to maintain in the future?